packages/create-miniapp-cli/src/helpers/installDependencies.ts

```
@@ -0,0 +1,47 @@
+import {
+    getUserPkgManager,
+    type PackageManager,
+} from '../utils/getUserPkgManager.js'
```

ESLint

[error] `../utils/getUserPkgManager.js` import should occur after import of `ora`

(import/order)